stable/prometheus/templates/server-configmap.yaml

```diff
@@ -20,6 +20,9 @@ data:
       alertmanagers:
       - kubernetes_sd_configs:
           - role: pod
+            namespaces:
+              names:
+                - {{ $root.Release.Namespace }}
```

Comment:
This should probably be optional.

Reply:
Do you have a reason it should be optional? On line 18 in this file it checks if alert manager is enabled. If it is enabled it will only ever be in the releases namespace?